avionics/common/include/radio.h

```diff
@@ -139,8 +138,11 @@ class RadioController {
                 // extra one, so we shouldn't respond twice again.
             }
 
+            read_count++;
+            if (read_count >= MAX_PACKETS_PER_RX_LOOP_) {
+                break;
+            }
```

Comment:
I feel like we could do with just one `xbee_.readPacket();`. Maybe combine `xbee_.readPacket();` and the condition in the `while` into a function and just put that in the `while`

Or turn the `while` into a `while(true)` with an if at the top checking if we should break or call `xbee_.readPacket();`.